### PR TITLE
Firefly-1491: add cutout support to data product viewer

### PR DESCRIPTION
**Pull request by robyww**
#### [Firefly-1491](https://jira.ipac.caltech.edu/browse/FIREFLY-1491): Add cutout support to data product viewer

euclid PR: https://github.com/IPAC-SW/irsa-ife/pull/336

#### Testing
- https://firefly-1491-cutout.irsakudev.ipac.caltech.edu/applications/euclid
  - Do a search, you should see a cutout option on the far left on the image toolbar
- https://fireflydev.ipac.caltech.edu/firefly-1491-cutout/firefly
  - to a obscore TAP search from CADC, choose the cutout option

### CODE REVIEW

File: src/firefly/js/ui/CutoutSizeDialog.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    );
+    DialogRootContainer.defineDialog(DIALOG_ID, popup);
+    dispatchShowDialog(DIALOG_ID);
+}
+
+
+
+
+function CutoutSizePanelPanel({cutoutParam,dataProductsComponentKey}) {
+
```
Comment (kpuriIpac) [inline src/firefly/js/ui/CutoutSizeDialog.jsx:37]: small typo here, should be `CutoutSizePanel`. And then in the call to this component on line 25.

**Review by kpuriIpac — APPROVED** [review]
Looks great, really good addition to the UI. Tested in both `Explore Regions` and `Inspect Sources`. Tried Degrees/Arcminutes/Arcseconds. Had some issues with the build breaking if I tried larger cutout sizes. I'm guessing that's due to the memory limit of the test build. 

Just the one typo in the component name in `CutoutSizeDialog.jsx` that I pointed out below.

**Comment by kpuriIpac** [comment]
Looks good for Euclid! I did an obscore TAP search on CADC, though, (single object m81) and don't see cutout option in the result view. Should I be trying something else?

**Comment by robyww** [comment]
Under the more dropdown you see and cutout option from CADC. I just tested it for m81, it works.

I think I will go ahead and merge

**Pull request merged**